apps/web/src/app/_components/reviews/role-card-preview.tsx

```diff
@@ -0,0 +1,97 @@
+"use client";
+
+import type { RoleType } from "@cooper/db/schema";
+import { cn } from "@cooper/ui";
+import { Card, CardContent, CardHeader, CardTitle } from "@cooper/ui/card";
+
+import { api } from "~/trpc/react";
+
+interface RoleCardPreviewProps {
+  className?: string;
+  reviewObj: RoleType;
+}
+
+export function RoleCardPreview({
+  className,
+  reviewObj,
+}: RoleCardPreviewProps) {
+  // ===== COMPANY DATA ===== //
+  const company = api.company.getById.useQuery({
+    id: reviewObj.companyId,
+  });
+
+  // ===== ROLE DATA ===== //
+  const role = api.role.getById.useQuery({ id: reviewObj.id });
+  const reviews = api.review.getByRole.useQuery({ id: reviewObj.id });
+
+  const yellowStar = (
```

Comment:
Let's see if you can add this to the SVG folder and import it and use it from there